src/internal.c

```diff
@@ -8704,6 +8753,8 @@ int SendKexDhReply(WOLFSSH* ssh)
                 privKey = (ecc_key*)WMALLOC(sizeof(ecc_key), heap,
                         DYNTYPE_PRIVKEY);
                 if (pubKey == NULL || privKey == NULL) {
+                    WFREE(pubKey, heap, DYNTYPE_PUBKEY);
```

Comment:
was this leaking before? both appear to be free'd here https://github.com/wolfSSL/wolfssh/pull/510/files#diff-1c4e2f5adfa1ad30618e78ff459b2c0758ecf34278459ad0a8d58db4fec622eaR8802